bin/dotd

```diff
@@ -435,13 +424,9 @@ def main
         end
 
         wait_for 'DTP to finish'
-        should_i 'reset DTP message to yes' do
-          DevelopersTopic.set_dtp 'yes'
-        end
-        should_i 'update Slack#infra-production topic' do
-          sha = GitHub.sha 'production'
-          InfraProductionTopic.set_dtp_commit(sha)
-        end
+        DevelopersTopic.set_dtp 'yes'
+        sha = GitHub.sha 'production'
+        InfraProductionTopic.set_dtp_commit(sha)
```

Comment:
For later: I wonder if updating the `#infra-production` topic should be part of our deploy process, rather than part of the DOTD script.  It seems like something we rarely check and rarely (ever?) do manual intervention for.